config/locales/provider_interface/provider_relationship_permissions.yml

```diff
@@ -8,6 +8,7 @@ en:
       description: View criminal convictions and professional misconduct
     view_diversity_information:
       description: View sex, disability and ethnicity information
+    no_provider_permitted: Neither organisation can do this
```

Comment:
for non setup permissions, was there no further content around you can't set it up if you have no permissions or is this the final version?

Reply:
In the prototype, the content is:
`Neither organisation can do this` if the user has not got manage_orgs permissions
`Neither organisation can do this - you can change permissions to allow it` if they do have permission

Since they will be redirected to setup in the latter case anyway, the content is quite simple. This is why we asked about the redirecting flow, as we are now relying on the redirect logic.